java/src/test/java/ai/rapids/cudf/UnaryOpTest.java

```diff
@@ -253,6 +253,24 @@ public void testLog() {
     }
   }
 
+  @Test
+  public void testLog2() {
+    try (ColumnVector dcv = ColumnVector.fromBoxedDoubles(DOUBLES_1);
+         ColumnVector answer = dcv.log2();
+         ColumnVector expected = forEach(dcv, doubleFun(n -> Math.log(n) / Math.log(2)))) {
+      assertColumnsAreEqual(expected, answer);
+    }
+  }
+
+  @Test
+  public void testLog10() {
+    try (ColumnVector dcv = ColumnVector.fromBoxedDoubles(DOUBLES_1);
+         ColumnVector answer = dcv.log10();
+         ColumnVector expected = forEach(dcv, doubleFun(Math::log10))) {
+      assertColumnsAreEqual(expected, answer);
+    }
+  }
+
```

Comment:
Can we add tests to see if calling `log()` on non double or empty columns does the right thing?